app/src/main/java/com/hedvig/app/feature/insurance/ui/ContractPillAdapter.kt

```diff
@@ -27,9 +25,11 @@ class ContractPillAdapter :
         fun bind(item: ContractModel) {
             binding.apply {
                 when (item) {
-                    is ContractModel.ContractType -> {
-                        text.text =
-                            text.context.getString(item.type.getStringIdForCard()).toUpperCase()
+                    is ContractModel.Address -> {
+                        item.currentAgreement?.asDanishHomeContentAgreement?.let { text.text = it.address.fragments.addressFragment.street.toUpperCase() }
```

Comment:
`.toUpperCase()` requires a locale to work properly.

Reply:
Also, I noticed that we're using `toUpperCase()` on a few text keys in this file, should the text keys themselves not just contain uppercased text?